dev-client/src/screens/SoilScreen/SoilScreen.tsx

```diff
@@ -94,18 +95,16 @@ export const SoilScreen = ({siteId}: {siteId: string}) => {
         alignItems="center">
         <Heading variant="h6">{t('soil.pit')}</Heading>
         {!projectSettings && (
-          <FormOverlaySheet
-            Header={
-              <Heading variant="h6">{t('soil.soil_preset.header')}</Heading>
-            }
+          <InfoSheet
```

Comment:
Is it by-design that this opens to 100% of the available height, even if the contents would allow 50%? 

Same question applies to EditDepthModal

<img src="https://github.com/user-attachments/assets/19b80b95-f5aa-4354-9d40-3bf704973d25" height="500">

Reply:
Yes, this was confirmed with @CourtneyLee333